tutorials/mongoose/findupdatedoc.md

```diff
@@ -0,0 +1,16 @@
+To make `findOneAndUpdate()` return the updated document, you need to use the `returnDocument` option.
+`returnDocument` has two options, `'before'` and `'after'`, with the default behavior being it gives the document before the update.
```

Comment:
"`returnDocument` has two possible values: 'before' and 'after'. The default behavior is 'before', which means returning the document as it was _before_ the update was applied."